commands/auth/2fa/disable.js

```diff
@@ -0,0 +1,21 @@
+const cli = require('heroku-cli-util')
+const co = require('co')
+
+function * run (context, heroku) {
+  let password = yield cli.prompt('Password', {hide: true})
+  let body = {password: password, two_factor_authentication: false}
+  let account = yield heroku.patch('/account', {body: body})
+  if (!account.two_factor_authentication) {
```

Comment:
@RasPhilCo could you make this more like the Go version https://github.com/heroku/cli/blob/master/auth.go#L344 by factoring out the status into library module and rerunning that from here so that we do not change behavior / output?